resources.html

```diff
@@ -57,7 +57,7 @@
 							<a class="page-scroll" href="./index.html">Home</a>
 						</li>
                         <li>
-                            <a class="page-scroll" href="http://54.191.129.39/">Get Started</a>
+                            <a class="page-scroll" href="http://54.191.129.39/" target="_blank">Get Started</a>
```

Comment:
@blengerich can you take a look at the new change? I just changed the IP address reference to "http://app.genamap.org/". But I wasn't sure if you meant writing a script that defined variables for each link and using those variables in place.